samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/OktaVerifyController.cs

```diff
@@ -228,13 +230,25 @@ public async Task<ActionResult> ChallengePoll()
             {
                 Refresh = pollResponse.Refresh ?? 4000,
                 ContinuePolling = pollResponse.ContinuePolling,
-                Next = "/Home/Index"
             };
 
             if (!pollResponse.ContinuePolling)
             {
-                ClaimsIdentity identity = await AuthenticationHelper.GetIdentityFromTokenResponseAsync(_idxClient.Configuration, pollResponse.TokenInfo);
-                _authenticationManager.SignIn(new AuthenticationProperties(), identity);                
+                switch (pollResponse.AuthenticationStatus)
+                {
+                    case AuthenticationStatus.Success:
+                        ClaimsIdentity identity = await AuthenticationHelper.GetIdentityFromTokenResponseAsync(_idxClient.Configuration, pollResponse.TokenInfo);
+                        _authenticationManager.SignIn(new AuthenticationProperties(), identity);
+                        pollViewModel.Next = "/Home/Index";
+                        break;
+                    case AuthenticationStatus.Failed:
```

Comment:
Since this status seems specific to push notifications, I suggest renaming it to something else., for example, `PushFailed`.